### PR TITLE
Accessibility evaluation

### PR DESCRIPTION
**Pull request by dodoburner**
In this pull request, my coding partner and I have checked my portfolio for proper accessibility,
 we have evaluated the following:

1. Page Title -  We have changed the[ page title](https://github.com/dodoburner/Portfolio/compare/main...accessability-evaluation#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L7-R7) so it is more descriptive 
2. Image Text Alternatives - We have added the appropriate [alt text ](https://github.com/dodoburner/Portfolio/compare/main...accessability-evaluation#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L23-R23)to the images
3. Headings - We have changed the[ headings](https://github.com/dodoburner/Portfolio/compare/main...accessability-evaluation#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L216-R216) appropriately 
4. Contrast Ratio - passed the check
5. Resize text - passed the check
6. Keyboard acces and visual controls - We have changed the [logo ](https://github.com/dodoburner/Portfolio/compare/main...accessability-evaluation#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L216-R216)so it can be accessed with the tab key
7. Forms, labels and errors - we don't have the required labels, but our portfolio design doesn't require them
8. Moving, flashing or blinking content - passed the check
9. Multimedia (video and audio) alternatives - passed the check
10. Basic structure check - We have added the header tags for the[ project titles ](https://github.com/dodoburner/Portfolio/compare/main...accessability-evaluation#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L63-R63)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,22 +47,22 @@ <h1 class="intro">
         <ul class="social-icons"> 
           <li><a class="social-media-icon" href=""><img src="images/social-icons/Vector.svg" alt="twitter link"></a></li>
           <li><a class="social-media-icon" href=""><img src="images/social-icons/Linkedinicon.svg" alt="linked-in link"></a></li>
-          <li><a class="social-media-icon" href=""><img src="images/social-icons/Vector(2).svg" alt=""></a></li>
+          <li><a class="social-media-icon" href=""><img src="images/social-icons/Vector(2).svg" alt="social-media link"></a></li>
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] please kindly add the name of the social media icon in the `alt`

File: index.html [hunk 2]
```diff
@@ -95,12 +95,12 @@ <h1 class="intro">
       </div>
 
       <div class="project-section project2">
-        <img class="project-img" src="images/SnapshootPortfolio(1).jpg" alt="">
-        <img class="project-img-desktop img2" src="images/desktop-images/img2.png" alt="">
```
Comment (tufoinnkuo10) [inline index.html:99]: - [ ] kindly add alt description to all your images. it help to know what the image is about.

File: index.html [hunk 3]
```diff
@@ -284,12 +284,15 @@ <h2>
...
-  
-      <input name="email" type="email" placeholder="email" required>
-  
+      <label for="name">Name</label>
+      <input id="name" name="name" type="text" maxlength="30" placeholder="name" required>
+      
+      <label for="email">Email</label>
+      <input id="email" name="email" type="email" placeholder="email" required>
+          
+      <label for="message">Message</label>
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] kindly follow the figma design.. the label is not found if the figma version

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @dodoburner 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Identified and fixed issues for good accessibility


Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 3 above)
Comment (dodoburner) [reply index.html]: But the task requires me to add a label, should I just leave the label but remove the value from it? @tufoinnkuo10

Comment (tufoinnkuo10) [reply index.html]: yes please you can do that... i know its necessary but the template you chose doesn't require it.

File: index.html (hunk 2 above)
Comment (dodoburner) [reply index.html:99]: Yes it is added this is the changed code that's why it is in red @tufoinnkuo10, the only place where it is not added is in the images used to style the page like the circles and arrows

**Review by vechicin — APPROVED** [review]
# Hi @dodoburner and @Lasten-Ernest ✋, 

Your project is complete! 🎯 🧠 💯 
There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights ⚡ 

- [x] Good PR title/description
- [x] Good job implementing the changes required by the previous reviewer

## Optional suggestions 📂 

- [ ] The previous reviewer accidentally missed this requirement, therefore it is [OPTIONAL]. However, if this was the first review this would be a [REQUIRED] change, so I strongly encourage you to implement it. According to the evaluation guidelines, we need to run your live demo through this [contrast checker](https://www.aditus.io/button-contrast-checker/). Some of the elements on your page are not passing the checks. I understand it is strange to be asked to follow a template that is not 100% accessible, but unfortunately, that is the crux of the matter. Consider running your live demo through the checks and fixing the ones that are not passing, even if they stray your design away from the template a little bit.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**